src/js/filterText.js

```diff
@@ -1,10 +1,10 @@
 export default function filterText(data) {
   let inputvalue = document.getElementById("text-filter").value;
-  //console.log(inputvalue);
-
+  console.log(inputvalue);
+  if (inputvalue) {
```

Comment:
I would expect this change to include indenting the next four lines, but it doesn't, resulting in inconsistent indentation.